Less Fleet Dress Brrr-et

juxjux9930 opened the pull request:
🆑 Jux
tweak: Fleet CAs and CNs now don't get the dress beret, and have to wear the full sailor getup.
/🆑

Aren't they so *cute* in their little sailor uniforms?

Spookerton commented on maps/torch/datums/uniforms_fleet.dm:
```diff
@@ -17,7 +17,10 @@
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/officer
 	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/command
-	dress_extra = list(/obj/item/material/sword/replica/officersword, /obj/item/clothing/head/beret/solgov/fleet/dress/command)
+	dress_extra = list(
+				/obj/item/material/sword/replica/officersword,
+				/obj/item/clothing/head/beret/solgov/fleet/dress/command,
+			)
```
```suggestion
	dress_extra = list(
		/obj/item/material/sword/replica/officersword,
		/obj/item/clothing/head/beret/solgov/fleet/dress/command
	)
```

Spookerton commented on maps/torch/datums/uniforms_fleet.dm:
```diff
@@ -54,7 +57,9 @@
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet
 
-	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet
+	dress_over = list(
+				/obj/item/clothing/suit/storage/solgov/dress/fleet
+			)
```
```suggestion
	dress_over = list(
		/obj/item/clothing/suit/storage/solgov/dress/fleet
	)
```

Spookerton commented on maps/torch/datums/uniforms_fleet.dm:
```diff
@@ -64,7 +69,10 @@
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/snco
-	dress_extra = list(/obj/item/material/sword/replica/officersword/pettyofficer, /obj/item/clothing/head/beret/solgov/fleet/dress)
+	dress_extra = list(
+				/obj/item/material/sword/replica/officersword/pettyofficer,
+				/obj/item/clothing/head/beret/solgov/fleet/dress
+			)
```
```suggestion
	dress_extra = list(
		/obj/item/material/sword/replica/officersword/pettyofficer,
		/obj/item/clothing/head/beret/solgov/fleet/dress
	)
```

Spookerton commented on maps/torch/datums/uniforms_fleet.dm:
```diff
@@ -85,7 +93,10 @@
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/officer
 	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/command
-	dress_extra = list(/obj/item/material/sword/replica/officersword, /obj/item/clothing/head/beret/solgov/fleet/dress/command)
+	dress_extra = list(
+				/obj/item/material/sword/replica/officersword,
+				/obj/item/clothing/head/beret/solgov/fleet/dress/command,
+			)
```
```suggestion
	dress_extra = list(
		/obj/item/material/sword/replica/officersword,
		/obj/item/clothing/head/beret/solgov/fleet/dress/command
	)
```

Spookerton commented on maps/torch/datums/uniforms_fleet.dm:
```diff
@@ -128,6 +139,10 @@
 	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet
+	dress_extra = list(
+				/obj/item/clothing/head/beret/solgov/fleet/dress
+			)
```
```suggestion
	dress_extra = list(
		/obj/item/clothing/head/beret/solgov/fleet/dress
	)
```

Spookerton commented on maps/torch/datums/uniforms_fleet.dm:
```diff
@@ -137,7 +152,10 @@
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/snco
-	dress_extra = list(/obj/item/material/sword/replica/officersword/pettyofficer, /obj/item/clothing/head/beret/solgov/fleet/dress)
+	dress_extra = list(
+				/obj/item/material/sword/replica/officersword/pettyofficer,
+				/obj/item/clothing/head/beret/solgov/fleet/dress
+			)
```
```suggestion
	dress_extra = list(
		/obj/item/material/sword/replica/officersword/pettyofficer,
		/obj/item/clothing/head/beret/solgov/fleet/dress
	)
```

Spookerton reviewed (changes requested):
just some style bits

juxjux9930 committed:
fleet dress beret go brr


Update maps/torch/datums/uniforms_fleet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/datums/uniforms_fleet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/datums/uniforms_fleet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/datums/uniforms_fleet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/datums/uniforms_fleet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/datums/uniforms_fleet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>

juxtaposed9930 commented:
Donezo Spook

SierraKomodo commented:
This feel somewhat arbitrary and pointless since there's other berets they have access to and can wear with the dress uniforms anyway, but sure why not.
